Yoast/wpseo-woocommerce
# Integrate with options framework

## Summary

This PR can be summarized in the following changelog entry:

* Integrate with the Yoast options framework

## Relevant technical choices:

* Decided to remove a couple of unused options for WooCommerce.
* Moved option checks that adds hooks to the callbacks for the hooks themselves, because I want to be sure the options has been loaded correctly.
* I converted the admin page to have used `Yoast_From` completely

## Test instructions

This PR can be tested by following these steps:

* On trunk set some values in the admin page for WooCommerce SEO
* Verify that these works: hide columns and go to the product page and make sure the yoast columns are still hidden
* Move the Woocommerce metabox above the yoast metabox
* Set the schema fields and verify they are in the schema output
* Checkout this branch and do the same.
* Also make sure fields are still saved on the admin page.

Fixes #418
